apps/design-system-docs/src/components/link-card-list/style.module.scss

```diff
@@ -0,0 +1,14 @@
+.link-card-list {
+	--ifm-list-item-margin: 0;
+
+	display: grid;
+	grid-template-columns: repeat(2, 1fr);
```

Comment:
Should we collapse this to single-column on smaller screens?

Reply:
Done in 29cb37f